### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by trymeouteh** [rafaelvcaetano/melonDS-android#33]
Please get this app in F-Droid and have it fetch the updates

**Comment by Fietoo** [comment]
What?

**Comment by rafaelvcaetano** [comment]
I will consider it once it's ready for the Play Store. Right now, there's no point since the current version is not very usable.

**Comment by ariebe9115** [comment]
Yes it is very laggy, even with Exynos 7884B and 3GB Ram i have an average of 40 FPS

**Comment by shabbirmahmud** [comment]
Duplicate of #15

**Comment by poudink** [comment]
Seems this has been available on the play store for quite a while now. Any update on F-Droid availability?

**Comment by dimaguy** [comment]
@rafaelvcaetano given the app is already on the play store, if you need I could help you out with getting it in f-droid

**Comment by Poussinou** [comment]
Since there is no tracker in the app [according to Exodus](https://reports.exodus-privacy.eu.org/en/reports/me.magnum.melonds/latest/), It shouldn't be too hard to include it in F-Droid. You can start by opening a RFP [here](https://gitlab.com/fdroid/rfp/-/issues).
